src/Screens/Logs/Log.tsx

```diff
@@ -161,6 +167,17 @@ const Log: FunctionComponent<LogProps> = ({ navigation, route }) => {
       });
     };
 
+    const deleteDiveLog = () => {
+      const url = `${config.API_ENDPOINT}/review/delete?review_id=${diveLog.review.id}`;
+      fetch(url, {
+        method: 'POST',
+        headers: {
```

Comment:
I think we should move this function to `redux/slices/dive-logs/api`

Reply:
fixed